bootstrap.py

```diff
@@ -291,6 +291,30 @@ def is_fips():
     return fips_status == "1"
 
 
+def get_rhsm_proxy():
+    """
+    Return the proxy server settings from /etc/rhsm/rhsm.conf as dictionary proxy_config.
+    """
+    rhsmconfig = SafeConfigParser()
+    rhsmconfig.read('/etc/rhsm/rhsm.conf')
+    proxy_options = [option for option in rhsmconfig.options('server') if option.startswith('proxy')]
+    proxy_config = {}
+    for option in proxy_options:
+        proxy_config[option] = rhsmconfig.get('server', option)
+    return proxy_config
+
+
+def set_rhsm_proxy(saved_proxy_config):
```

Comment:
Can you name the var in the function definition differently than the one you store the data in? Otherwise pylint barfs:

`************* Module bootstrap
W:307,19: Redefining name 'saved_proxy_config' from outer scope (line 1279) (redefined-outer-name)`